virttest/utils_misc.py

```diff
@@ -4538,3 +4538,23 @@ def _wait_for_commands(bg_jobs, start_time, timeout):
         bg_job.result.duration = time.time() - start_time
 
     return True
+
+
+def get_pid(name, session=None):
+    """
+    Get pid by process name
+
+    :param name: Name of the process to retrieve its pid
+    :param session: ShellSession object of VM or remote host
+    :return: Pid of the process or raise exception if process is not available
+    """
+    cmd = "ps -ef | grep %s | grep -v grep" % name
```

Comment:
let us wrap the name in quotes,
`cmd = "ps -ef | grep '%s' | grep -v grep" % name`

Reply:
fixed